### ISSUE TITLE
Facebook logins stopped working on Android devices

### ISSUE THREAD

**Issue by jobinkurianm** [firebase/firebaseui-web#1068]
### Describe your environment

Issue noticed in mobile web :  Android devices + chrome browser

Operating system: Android 12 
Chrome version : 120.0.6099.230
Configuration used :    'signInFlow': 'popup',
Tried different firebase-ui-auth versions 6.1.0, 6.0.0 with  firebase JS SDK v9 & v10 compat, but no luck.

### Describe the problem

In mobile devices Facebook login is not working , No error message/ warnings displayed.
Created exactly same project & updated firebase configurations:- https://fir-ui-demo-84a6c.firebaseapp.com/

#### Steps to reproduce:

Create exactly same project - https://fir-ui-demo-84a6c.firebaseapp.com/ and try it on  chrome browser in an android device
( 'signInFlow': 'popup')
Click facebook login , after login from facebook  it will just land on app-login page. without logging in.

#### Additional information

Working fine in desktop browsers and IOS Devices .
Also noticed,  if we changed  'signInFlow': 'redirect', it will work first time , (second time we need to clear the cookie to make it work)
```

**Comment by jhuleatt** [comment]
Hi @jobinkurianm, I'll update the Facebook setup for our live demo so I can see if I can reproduce this issue. In the meantime, I wonder if this might be an issue with the underlying Firebase JS SDK - see https://github.com/firebase/firebase-js-sdk/issues/7958 for more info.

**Comment by jobinkurianm** [comment]
@jhuleatt Yes, Looks like issue with firebase SDK itself . Tried direct implementation using 'signInWithPopup', 
but getting error 'Firebase: Error (auth/popup-closed-by-user)'

**Comment by jhuleatt** [comment]
Thanks for confirming. Along with the previous issue I mentioned (https://github.com/firebase/firebase-js-sdk/issues/7958), it's probably worth keeping an eye on this issue too: https://github.com/firebase/firebase-js-sdk/issues/7807.

**Comment by cptnoblivious** [comment]
Having this same issue on our production site. Worse, both signInWithPopup() and signInWithRedirect() are affected in both the 120.x and beta 123.x versions of Chrome, specifically on Android. Literally all other browsers on all other platforms work fine - including Chromium-based browsers on Android such as Adblock Browser.

**Comment by Ishan5hrestha** [comment]
did anybody solve this?

**Comment by jacquesgiraudel** [comment]
Same problem for me, works fine on Desktop and iOS device but not on Android: signInWithPopup returns auth/popup-closed-by-user, signinWithRedirect does not work (getRedirectResult(getAuth()) returns null after redirect).

**Comment by CT-HTrieu** [comment]
Same problem for me, works fine on Desktop device but not on Browser mobile android and ios: signInWithPopup returns auth/popup-closed-by-user, signinWithRedirect does not work (getRedirectResult(getAuth()) returns null after redirect).

**Comment by CT-HTrieu** [comment]
> Same problem for me, works fine on Desktop and iOS device but not on Android: signInWithPopup returns auth/popup-closed-by-user, signinWithRedirect does not work (getRedirectResult(getAuth()) returns null after redirect).

do you have any solution??

**Comment by NhienLam** [comment]
Hi everyone, could you check if your [authDomain](https://github.com/firebase/firebase-js-sdk/blob/beaa4dffb7f48cb12ccc6c1d1b7cdc9c3605fc04/packages/auth/demo/src/sample-config.js#L18) matches the domain you're using? If not, please try updating your authDomain to match the domain you're using and see if the flow works?

For example, if your web domain is "example.com",
your `authDomain` should also be "example.com"

Also ensure that you've add your domain to the list of authorized domains in the [Firebase console](https://console.firebase.google.com/project/_/authentication/settings) (Authentication -> Settings).

We're investigating why this is occurring only on Android devices and Facebook provider. Thank you for your patience.

**Comment by dmitry-yudakov** [comment]
We ended up implementing [Facebook native login for the web](https://developers.facebook.com/docs/facebook-login/web) combined with [Firebase manual sigh-in flow](https://firebase.google.com/docs/auth/web/facebook-login#advanced:-handle-the-sign-in-flow-manually) (signInWithCredential created with accessToken from fb SDK).

**Comment by jacquesgiraudel** [comment]
Thank you for the advice, I note the solution

**Comment by dackers86** [comment]
Thank you for taking the time to submit this bug.

A new release is planned shortly, which may resolve this request. 

In the meantime, we will close this issue, if this is not resolved in the new version - please feel free to create a new issue.
